catalogs/catalog-hadoop/src/main/java/org/apache/gravitino/catalog/hadoop/HadoopCatalogOperations.java

```diff
@@ -140,20 +177,20 @@ public void initialize(
 
   private void initAuthentication(Map<String, String> conf, Configuration hadoopConf) {
     AuthenticationConfig config = new AuthenticationConfig(conf);
-    String authType = config.getAuthType();
 
-    if (StringUtils.equalsIgnoreCase(authType, AuthenticationMethod.KERBEROS.name())) {
-      hadoopConf.set(
-          HADOOP_SECURITY_AUTHENTICATION,
-          AuthenticationMethod.KERBEROS.name().toLowerCase(Locale.ROOT));
-      UserGroupInformation.setConfiguration(hadoopConf);
-      try {
-        KerberosClient kerberosClient = new KerberosClient(conf, hadoopConf);
-        File keytabFile = kerberosClient.saveKeyTabFileFromUri(catalogInfo.id());
-        this.kerberosRealm = kerberosClient.login(keytabFile.getAbsolutePath());
-      } catch (IOException e) {
-        throw new RuntimeException("Failed to login with Kerberos", e);
-      }
+    if (config.isKerberosAuth()) {
+      this.kerberosRealm =
+          initKerberos(
+              conf,
+              hadoopConf,
+              NameIdentifier.of(catalogInfo.namespace(), catalogInfo.name()),
+              true);
+    } else if (config.isSimpleAuth()) {
+      UserGroupInformation u =
+          UserGroupInformation.createRemoteUser(PrincipalUtils.getCurrentUserName());
```

Comment:
Why do we use `RemoteUser`?

Reply:
So in simple mode, how can I create an UGI?

Reply:
UserGroupInformation.getCurrentUser.